client/src/main/proto/app/cash/backfila/service.proto

```diff
@@ -76,6 +83,7 @@ message CreateBackfillResponse {
 // Creates a backfill under the calling service and starts it.
 message CreateAndStartBackfillRequest {
   optional CreateBackfillRequest create_request = 1;
+  optional string variant = 2;
```

Comment:
why is this not part of `CreateBackfillRequest` ?

Reply:
It could be. I put it here to start with because the fields in `CreateBackfillRequest` seemed to moreso be properties of the run, rather than the service + variant that registered it.

Reply:
It will need to be part of both I think?

Reply:
It won't as the CreateBackfillRequest is used by the UI rather than the App